pkg/test/env/istio.go

```diff
@@ -108,7 +108,11 @@ func getDefaultIstioTop() string {
 	if err != nil {
 		panic(err)
 	}
-	idx := strings.Index(current, "/src/istio.io/istio")
+	substr := "/src/istio.io/istio"
+	if runtime.GOOS == "windows" {
```

Comment:
Can you do something with `path/filepath`? Perhaps using `Join()`?
https://golang.org/pkg/path/filepath/

Reply:
@jwendell filepath used